de.dashup.application/src/main/webapp/components/dashup-component.js

```diff
@@ -0,0 +1,18 @@
+import {LitElement, html, css} from "https://unpkg.com/lit-element@2.1.0/lit-element.js?module";
+export class DashupComponent extends LitElement {
+
+    static get properties() {
+        return {
+            name: {type: String},
+            layout: {type: Object}
+        };
+    }
+
+    constructor() {
+        super();
+        new CSSStyleSheet().replace("@import url('https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css')").then((sheet) => {
```

Comment:
You might also bind the files `theme.style.css` and the respective theme, so that within our widgets, the theme is also applied.

Reply:
Theming and making components resizable is put on the list of tasks for me ;)